vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -39,7 +39,7 @@ These steps use the AWS console to create the cluster. Experienced AWS users may
    8. Name your role (you’ll need this later), e.g. YOUR-EKS-ROLE
    9. Click `Create role`
 
-3. Refine the role to enable `Node Group` management and add elastic load balancer (ELB) access so the cluster can deploy the ELB for NGINX
+3. Refine the role to enable `Node Group` management and add Elastic Load Balancer (ELB) access so the cluster can deploy the ELB for NGINX
```

Comment:
should we fix this in the wiki? https://github.com/entando/entando-docs/wiki/Terminology

Reply:
@Lyd1aCla1r3 yeah, I think so. "load balancing" is a generic term and therefore lowercase. "Elastic Load Balancing" is the actual Amazon feature (so I should fix the name here) and capitalized in their docs - https://aws.amazon.com/elasticloadbalancing/getting-started/?nc=sn&loc=4